.github/workflows/build.yml

```diff
@@ -80,12 +82,23 @@ jobs:
         run: make check
       - name: Run distribution tests
         run: make distcheck
+      - name: Meson configure
+        run: |
+          meson setup build \
+            -Dbuild-tests=true \
+            -Dbuild-manual=true
+      - name: Meson build and generate man pages
+        run: ninja -C build
+      - name: Meson tests
+        run: cd build && meson test
+      - name: Meson distribution tests
+        run: cd build && meson dist
 
   build-debian:
     name: Debian
     runs-on: ubuntu-22.04
     container:
-      image: debian:oldstable
+      image: debian:bookworm
```

Comment:
For the record, the main reason to hold this on oldstable (bullseye) is for building against OpenSSL 1.1 for the sake of the DHX UAM.

But you were saying that the version of Meson on bullseye is too ancient to be usable, didn't you?

Reply:
From what I've seen, you can indeed compile the DHX library with the warning, but it will crash when you attempt to authenticate from a classic Mac OS client. Because OpenSSL will reject the weak crypto. It's a quite insidious fail state.